tools/code-generation/generator/pom.xml

```diff
@@ -128,5 +128,20 @@
             <artifactId>commons-lang</artifactId>
             <version>2.5</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.smithy</groupId>
+            <artifactId>smithy-jmespath</artifactId>
+            <version>1.24.0</version>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.smithy</groupId>
+            <artifactId>smithy-codegen-core</artifactId>
```

Comment:
why do we need explicitly need these dependencies instead of just implicitly getting them through jmespath?

Reply:
I am not sure I understand this comment. Do you mean try
<dependency>
            <groupId>software.amazon.smithy</groupId>
 </dependency>

Reply:
if you remove the `smithy-codegen-core` and `smithy-rules-engine` jmespath will still bring the dependencies in, you can remove a direct dependency on these as we are only interested in bringing in jmes path and whatever smithy deps it brings in, thats fine, we dont care what version they are because we dont use them directly

Reply:
ack. SO, look slike only smithy-codegen-core and smithy-jmespath are needed